### ISSUE TITLE
Implement different fonts for header and body text

### ISSUE THREAD

**Issue by hauwy-ismyl** [meshery/meshery.io#1474]
### Current State
Currently, meshery.io uses a single font for both header and body text which could negatively impact the overall aesthetics and readability. This issue is to make the website more visually appealing and user-friendly.

### Desired State 
The header text should have one, and the body text should have a different font for improved readability and aesthetics (see guide below). 

![Screenshot 2023-10-24 010425](https://github.com/meshery/meshery.io/assets/148660658/9064f009-1306-4f69-89c3-9878494cfac5)

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

**Comment by leecalcote** [comment]
I think this reinforces our brand guide and is a step in the right direction - meshery.io/brand

**Comment by Priyanshuthapliyal2005** [comment]
can you provide me the folder details in which i work on the issue
@leecalcote @hauwy-ismyl

**Comment by trinayreddy4** [comment]
@leecalcote can i work on this?

**Comment by leecalcote** [comment]
Yes, please.

**Comment by sharvarii11** [comment]
hi, @trinayreddy4 r u still working on this ?

**Comment by abhijeetgauravm** [comment]
Any updates here @trinayreddy4 ?

**Comment by trinayreddy4** [comment]
@abhijeetgauravm yeah i was busy with my end sem college exams would be discussing about it in the evening web meeting...

**Comment by abhijeetgauravm** [comment]
@sharvarii11 do u want to address this?

**Comment by omesh-omg** [comment]
@abhijeetgauravm is there somebody working on it or is it done?
if not can you assign this to me?

**Comment by saurabh100ni** [comment]
@omesh-omg you still up for this issue?

**Comment by omesh-omg** [comment]
Yes

**Comment by sudhanshutech** [comment]
@omesh-omg ?

**Comment by omesh-omg** [comment]
> @omesh-omg ?
Hello @sudhanshutech

**Comment by Vanshikadahaliya** [comment]
Is there anyone working on this? I want to work on this issue.
